api/controllers/services/ckan/agencyService.js

```diff
@@ -9,8 +9,8 @@ class AgencyService extends AbstractService {
         this.agencyResourceId = this.config.agency_resource;
```

Comment:
Leaving all the ckan services as it is. added another folder scosv2 and copied the updated versions of the services there.